src/vt/vrt/collection/balance/node_lb_data.cc

```diff
@@ -273,6 +273,9 @@ getRecvSendDirection(elm::CommKeyType const& comm) {
   case elm::CommCategory::CollectiveToCollectionBcast:
   case elm::CommCategory::LocalInvoke:
     return std::make_pair(ElementIDType{}, ElementIDType{});
+  case elm::CommCategory::WriteShared:
+  case elm::CommCategory::ReadOnlyShared:
+    return std::make_pair(ElementIDType{}, ElementIDType{});
```

Comment:
This is no different that previous cases:
```suggestion
  case elm::CommCategory::CollectiveToCollectionBcast:
  case elm::CommCategory::LocalInvoke:
  case elm::CommCategory::WriteShared:
  case elm::CommCategory::ReadOnlyShared:
    return std::make_pair(ElementIDType{}, ElementIDType{});
```